packages/common/src/hooks/useIsScreen/useIsMediumScreen.test.ts

```diff
@@ -0,0 +1,26 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { setScreenSize_ONLY_FOR_TESTING, TestingProvider } from '../../utils';
+import { useIsMediumScreen } from './useIsMediumScreen';
+
+describe('useIsMediumScreen', () => {
+  it('Returns true when the screen is less than 1440', () => {
+    setScreenSize_ONLY_FOR_TESTING(1339);
+    const { result } = renderHook(useIsMediumScreen, {
+      wrapper: TestingProvider,
+    });
+    const { current } = result;
+
+    expect(current).toBe(true);
+  });
+
+  it('Returns false when the screen is greater than 1440', () => {
+    setScreenSize_ONLY_FOR_TESTING(1440);
```

Comment:
:nit this feel real pedantic sorry. The test says _greater than_ 1440 and the screen size _is_ 1440

Reply:
33dfd46 😁